hangman.rb

```diff
@@ -0,0 +1,31 @@
+random_words = ["camera", "photo", "picture", "landscape", "light", "shadow", "lens", "portrait"]
+#the number is one more
+random_num = rand(0..7).to_i
```

Comment:
I think this can be simplified to `rand(7)`. That will return you a value from 0 through 6.